strawberry/schema/schema_converter.py

```diff
@@ -653,3 +664,11 @@ def is_type_of(obj: Any, _info: GraphQLResolveInfo) -> bool:
             return is_type_of
 
         return None
+
+    def validate_same_type_definition(
+        self, name: str, type_definition: StrawberryType, cached_type: ConcreteType
+    ) -> None:
+        if cached_type.definition != type_definition:
+            print(cached_type.definition)
+            print(type_definition)
```

Comment:
We probably don't want to keep those prints. If the information they're printing is relevant they should be part of the exception :) What do you think?

```suggestion
```

Reply:
Indeed, this was just for debugging. I'll remove it:)